packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -1037,6 +1038,23 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 			}
 		}
 
+		async function onCopy(event: any) {
+			const copiedContent = editor.selection.getContent();
+
+			// We need to remove timestamps from the image URLs while copying
+			// because some offline edtors do not show the image if there is
+			// an extra ?t= in it's path.
+			// Related to - https://github.com/laurent22/joplin/issues/4602
+			const removeTimestampFromUrl = (url: string) => {
+				const index = url.lastIndexOf('?t=');
+				return index === -1 ? url : url.substr(0, index);
+			};
+
+			const updatedContent = HtmlUtils.replaceImageUrls(copiedContent, removeTimestampFromUrl);
+			clipboard.writeHTML(updatedContent);
+			if (event) event.preventDefault();
```

Comment:
Is "event" not defined sometimes, and if so in which context?

Reply:
Ah, that's a mistake. Removed the condition.